tests/src/test/java/alluxio/server/tieredstore/TieredStoreIntegrationTest.java

```diff
@@ -236,4 +242,38 @@ public void promoteBlock() throws Exception {
       }
     }, WAIT_OPTIONS);
   }
+
+  /**
+   * With setting of "StorageDir1, StorageDir2", loading a block from UFS that is bigger than
+   * the available space of StorageDir1 should land in StorageDir2 using GreedyAllocator.
+   * https://github.com/Alluxio/alluxio/issues/8687
+   */
+  @Test
+  public void greedyAllocator() throws Exception {
+    mLocalAlluxioClusterResource.stop();
+    int fileLen = 8 * Constants.MB;
+    mLocalAlluxioClusterResource
```

Comment:
Can't we use the `Config` annotation to update configuration for a particular test? Like: https://github.com/Alluxio/alluxio/blob/master/tests/src/test/java/alluxio/client/cli/fs/command/PersistCommandTest.java#L362

Then we don't have to stop and start the cluster, which is not cheap.

Reply:
that's where I started with `config` annotation. Then I realized it requires compile-time **String constant**, in here a lot values I'm setting are derived (e.g.,  tmp dir and even the tiered property keys are derived from template)